notebooks/Manage Map Areas with Group and Index.ipynb

```diff
@@ -0,0 +1,456 @@
+    "More details on ArcGIS API for Python 2.4: https://www.esri.com/arcgis-blog/products/api-python/announcements/whats-new-in-arcgis-api-for-python-2-4-0-blog/\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "### Setup \n",
+    "\n",
+    "The script utilizes three items that will ***need to be created*** and configured in your portal \n",
+    "\n",
+    "#### Hosted Feature Layer \n",
+    "##### `Manage Map Areas`\n",
+    "- Setup Notes: Download the zipped fgdb and add it as an item to your portal.  The hosted feature layer and table below are both included.\n",
+    "\n",
+    "    - File hosted on ArcGIS Online: https://www.arcgis.com/home/item.html?id=2ab1be89349e4e61be674fc3e3cbfcdf\n",
+    "\n",
+    "    ##### `Manage Map Area Index`\n",
+    "    - Purpose: The polygon hosted feature layer stores map area boundries and processing details.  You can specify if the map area applied to all web map in the group, or a specific web map item id\n",
+    "    - Configuraton: Add your own map area polygons and configure refresh details.  You can specify the maps area to be applied to \"ALL\" web maps or a single web map item id.  You can also set map areas to \"Active\" or not.  Properties included in https://developers.arcgis.com/python/latest/api-reference/arcgis.map.toc.html#offlinemapareamanager can be configured  \n",
+    "\n",
+    "    ##### `Manage Map Area Processing Log`\n",
+    "    - Purpose: Log table where processing details are stored, this table is used for reporting.\n",
+    "    - Configuration: None needed, the processing script is designed to add rows to this table, but can be configured to truncate for each run as well.\n",
+    "      \n",
+    "#### Group:\n",
+    "##### `Manage Map Areas`\n",
+    "- Purpose: Group should include web maps that will have their map areas processed.  The owner and members of this group will receive email summary of processing results \n",
+    "- Configuration: Any web map that you own and would like to process map areas for should be added to this group. Any users that you would like to receive processing notification should be included in this group.  This notification is optional\n",
+    "\n",
+    "#### After items are created\n",
+    "Copy paste the item ids for `Manage Map Areas` HFL and `Manage Map Areas` group into the `Get Manage Map Areas HFL and Group` section below\n",
+    "\n",
+    "--- \n",
+    "\n",
+    "#### Authentication / Security \n",
+    "\n",
+    "The notebook below uses built-in user (`gis = GIS(\"home\")`), which is the account you're logged onto your portal with.  \n",
+    "\n",
+    "- For information on different authentication schemes see:\n",
+    "    - https://developers.arcgis.com/python/guide/working-with-different-authentication-schemes/\n",
+    "- For protecting your credentials see:\n",
+    "    - https://developers.arcgis.com/python/guide/working-with-different-authentication-schemes/#protecting-your-credentials\n",
```

Comment:
Nice! The markdown here is super helpful Jake.

Reply:
I couldn't really find an enterprise instance that had a notebook and an account that I could update to point to my email - so I didn't do any testing on the email feature for enterprise.  I suppose we could add a note that additional configuration may be need to enable the feature on enterprise and add the link @lau13820  found - thoughts @justincolville ?

Reply:
>I suppose we could add a note that additional configuration may be need to enable the feature on enterprise and add the link @lau13820 found

I think thats a good idea.  Doesn't hurt to include the comment and link